docs/lib/predictions/fragments/android/getting-started.md

```diff
@@ -0,0 +1,171 @@
+
+Create the file `amplifyconfiguration.json`:
+```
+touch amplifyconfiguration.json
+```
+
+Copy the contents over and update the values for the specific predictions method you are looking to use:
+```
+{
+    "UserAgent": "aws-amplify-cli/2.0",
+    "Version": "1.0",
+    "Predictions": {
+        "plugins": {
+            "awsPredictionsPlugin": {
+                "defaultRegion": "us-west-2",
+                 "identify": {
+                    "identifyText": {
+                        "format": "ALL",
+                        "region": "us-west-2",
+                        "defaultNetworkPolicy": "auto"
+                    },
+                    "identifyEntities": {
+                        "maxEntities": "0",
+                        "celebrityDetectionEnabled": "true",
+                        "region": "us-west-2",
+                        "defaultNetworkPolicy": "auto"
+                    },
+                    "identifyLabels": {
+                        "region": "us-west-2",
+                        "type": "LABELS",
+                        "defaultNetworkPolicy": "auto"
+                    }
+                },
+                "convert": {
+                    "translateText": {
+                        "targetLang": "zh",
+                        "sourceLang": "en",
+                        "region": "us-west-2",
+                        "defaultNetworkPolicy": "auto"
+                    },
+                    "speechGenerator": {
+                        "voice": "Salli",
+                        "language": "en-US",
+                        "region": "us-west-2",
+                        "defaultNetworkPolicy": "auto"
+                    }
+                },
+                "interpret": {
+                    "interpretText": {
+                        "region": "us-west-2",
+                        "defaultNetworkPolicy": "auto"
+                    }
+                }
+            }
+        }
+    }
+}
+```
+Add both the `amplifyconfiguration.json` and the `awsconfiguration.json` to your project inside the `res/raw` directory.
+
+## Configure your application
+
+Add the following dependencies to your **app**'s' `build.gradle` file and click "Sync Now" when prompted:
+
+```gradle
+dependencies {
+    implementation 'com.amplifyframework:core:0.10.0'
+    implementation 'com.amplifyframework:aws-predictions:0.10.0'
+    implementation 'com.amazonaws:aws-android-sdk-mobile-client:2.16.+'
+}
+```
+
+Add the following compile options in the same file to support the Java 8 features that Amplify uses:
+
+```gradle
+android {
+  compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+}
+```
+
+## Initialize Amplify
+
+Add the following code to your `MainActivity#onCreate` method (ideally this would go in your Application class but this works for getting started quickly):
```

Comment:
Our project setup guide will have them create an `Application` class, so we can change this to alter that. Since Auth is in master, can we use that instead of `AWSMobileClient`? cc @TrekSoft

Reply:
Yea this portion of documentation will require an update after Auth category is finished.